### DISCUSSION TITLE
Git locked files

### DISCUSSION THREAD

**Discussion by Gleb0n** [community/community#56108] [Actions]
### Select Topic Area

Question

### Body

I working on Unreal Engine 5 project by using GitHub to version control. And while working , there was a problem with commiting changes on repository. When I press 'commit' appear error log that tell "A lock file already exists in the repository, which blocks this operation from completing." and I have no idea what i'm doing wrong. I already check locked list by using "git lfs lock", by this command show just an empty list. I will be grateful for any information about what might be causing this problem.

**Comment by ccalvocm** [comment]
Greetings @Gleb0n 

It sounds like you might be running into a Git LFS (Large File Storage) issue. Git LFS is used to manage large files in Git repositories, and it uses lock files to prevent multiple users from modifying the same file at the same time.

Here are some steps you can take to resolve the issue:

1. Check if there are any lock files in the repository by running the command git lfs locks. If there are any lock files, you can remove them by running the command git lfs unlock <filename>.
2. If there are no lock files, try running the command git lfs prune. This will remove any unreferenced Git LFS objects from the repository.
4. If neither of the above steps work, you can try deleting the .git/lfs/locks directory in your local repository. This will remove all lock files, but it should only be done if you are sure that no one else is currently modifying the files in the repository.

If none of the above steps work, you can try resetting your Git LFS configuration by running the command git lfs uninstall followed by git lfs install.

Once you have resolved the lock file issue, you should be able to commit your changes to the repository. If you continue to have issues, you may want to reach out to the Unreal Engine community or support team for further assistance.
